vortex-serde/src/layouts/write/writer.rs

```diff
@@ -107,53 +104,42 @@ impl<W: VortexWrite> LayoutWriter<W> {
         }
 
         if let Some(batches) = self.column_chunks.get_mut(column_idx) {
-            batches.row_offsets.extend(row_offsets);
+            batches.batch_row_offsets.push(row_offsets);
             batches.batch_byte_offsets.push(byte_offsets);
         } else {
             self.column_chunks
-                .push(BatchOffsets::new(row_offsets, vec![byte_offsets]));
+                .push(BatchOffsets::new(vec![row_offsets], vec![byte_offsets]));
         }
 
         Ok(())
     }
 
     async fn write_metadata_arrays(&mut self) -> VortexResult<Layout> {
```

Comment:
this does not write any thing, unlike some other methods of this type. a better name would be `build_layout` imo

Reply:
yeah, because we don't have any metadata anymore. They will be added back.